src/main/java/frc/robot/commands/teleop/vision/VisionRun.java

```diff
@@ -1,60 +1,51 @@
-package frc.robot.commands.teleop;
+package frc.robot.commands.teleop.vision;
 
-import frc.robot.subsystems.Ringlight;
 import edu.wpi.first.wpilibj2.command.CommandBase;
-import frc.robot.Constants;
-import frc.robot.controls.Driver;
 import frc.robot.Robot;
 
-public class TeleopRinglight extends CommandBase
+public class VisionRun extends CommandBase
 {
   @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
-  private final Ringlight ringlight;
-  private boolean on = false;
+  private Robot robot;
 
-  public TeleopRinglight(Robot robot)
+  public VisionRun(Robot robot)
   {
-    ringlight = robot.ringlight;
-    addRequirements(ringlight);
+    this.robot = robot;
+    addRequirements(robot.vision);
   }
 
   // Called when the command is initially scheduled.
   @Override
   public void initialize()
   {
-    ringlight.off();
+
   }
 
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute()
   {
-    if (Driver.getSteerButtonPressed(Constants.TOGGLE_RINGLIGHT))
```

Comment:
Could make a function called `Driver.toggleRinglight()` that returns a `boolean`.